server/store/store.go

```diff
@@ -24,6 +24,7 @@ type Store interface {
 	UserHasConnected(mmUserID string) (bool, error)
 	GetUserConnectStatus(mmUserID string) (*storemodels.UserConnectStatus, error)
 	GetHasConnectedCount() (int, error)
+	//@withTransaction
```

Comment:
Instead of annotating the interface, is it possible to annotate the implementation instead? This is an implementation detail, doesn't apply to all implementations of the interface, and doesn't concern the caller anyway. Plus I'd hate to see a refactor accidentally move a comment away from the "line" after.

Reply:
Totally agree with the reasoning here. I'll give it a try as it's a sizeable change